<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="7" hostname="LAPTOP-2KQV9F30" failures="0" tests="1" name="Default test" time="5.285" errors="0" timestamp="2025-05-20T20:05:14 CDT">
  <testcase classname="tests.StackTest" name="verifyUserIsInStackPage" time="0.423"/>
  <testcase classname="tests.StackTest" name="verifyRunButtonWithoutEnteringCode" time="0.0">
    <ignored/>
  </testcase> <!-- verifyRunButtonWithoutEnteringCode -->
  <testcase classname="tests.StackTest" name="validPythonCodeforTryEditor" time="0.0">
    <ignored/>
  </testcase> <!-- validPythonCodeforTryEditor -->
  <testcase classname="tests.StackTest" name="verifyStacksNavigation" time="0.0">
    <ignored/>
  </testcase> <!-- verifyStacksNavigation -->
  <testcase classname="tests.StackTest" name="test_Application_Practice_Questions_Link_Stack_Bug" time="0.0">
    <ignored/>
  </testcase> <!-- test_Application_Practice_Questions_Link_Stack_Bug -->
  <testcase classname="tests.StackTest" name="Signout" time="0.0">
    <ignored/>
  </testcase> <!-- Signout -->
  <testcase classname="tests.StackTest" name="VerifyTryHere" time="0.0">
    <ignored/>
  </testcase> <!-- VerifyTryHere -->
  <testcase classname="tests.StackTest" name="InvalidPythonCodeforTryEditor" time="0.0">
    <ignored/>
  </testcase> <!-- InvalidPythonCodeforTryEditor -->
</testsuite> <!-- Default test -->
